src/Raven.Studio/typescript/models/wizard/continueSetup.ts

```diff
@@ -3,22 +3,36 @@
 import extractNodesInfoFromPackageCommand = require("commands/wizard/extractNodesInfoFromPackageCommand");
 import fileImporter = require("common/fileImporter");
 
+interface configurationNodeInfo  {
+    publicServerUrl: string;
+    serverUrl: string;
+    tag: string;
+}
+
 class continueSetup {
 
     importedFileName = ko.observable<string>();
     hasFileSelected = ko.observable(false);
-    nodesInfo = ko.observableArray<Raven.Server.Web.System.ConfigurationNodeInfo>([]);
-    
+    nodesInfo = ko.observableArray<configurationNodeInfo>([]);
     
     zipFile = ko.observable<string>();
     nodeTag = ko.observable<string>();
     serverUrl = ko.observable<string>();
     
+    isZipFileSecure: KnockoutComputed<boolean>;
```

Comment:
what do we keep in this variable? The name is confusing.

Reply:
@Danielle9897

Reply:
**in the 'continue setup flow',** 
depending on the content of this variable we go to either ep:

if secure => we continue to ep:  `/setup/continue`
if not-secure => we continue to ep: `/setup/continue/unsecured`

The current logic by which the variable gets set is:
we parse the zip file provided by the user (by calling ep `/setup/continue/extract`)
and we get the list of nodes in the zip,
and logic is:
* nodes from Unsecure zip contain **ServerUrl**
* nodes from Secure zip contain **PublicServerUrl**
            
so that is how to know if secure/not-secure zip